src/core/chunked_stream.js

```diff
@@ -35,9 +35,16 @@ var createPromiseCapability = sharedUtil.createPromiseCapability;
 var isInt = sharedUtil.isInt;
 var isEmptyObj = sharedUtil.isEmptyObj;
 
-var ChunkedStream = (function ChunkedStreamClosure() {
-  function ChunkedStream(length, chunkSize, manager) {
-    this.bytes = new Uint8Array(length);
+/* If the length of stream is less than ALLOCATE_NO_CHUNKS_SIZE then use
+ChunkedStreamContinuous which works with single block of memory */
```

Comment:
Nit: incorrect comment style, please use `//` instead. And please remember to always add a period at the end of sentences (applies elsewhere in this patch as well). E.g. like this:
```javascript
// If the length of stream is less than ALLOCATE_NO_CHUNKS_SIZE then use
// ChunkedStreamContinuous which works with single block of memory.
```